content/ru/continuous-delivery.md

```diff
@@ -9,27 +9,27 @@ tags: ["methodology", "application", ""]
 
 Непрерывная доставка (Continuous Delivery), часто сокращенно называемая CD, — это набор практик, 
 при которых изменения кода автоматически развертываются в приемочное окружение 
-(или, в случае непрерывного развертывания, в production).
+— или, в случае [непрерывного развертывания](/continuous-deployment/) (Continuous Deployment), в production.
```

Comment:
While I agree with this link, to have it here, we need to change the original (English) version first.
```suggestion
— или, в случае непрерывного развертывания, в production.
```